.github/workflows/pypi-release.yaml

```diff
@@ -4,8 +4,8 @@ on:
     types:
       - published
   push:
-    branches:
-      - master
+    tags:
+      - 'v*'
```

Comment:
I think this will result in duplicate runs i.e. one for the push event and the release event... Do we want this duplication?

Reply:
I think that's okay (intended even), as long as we only upload to PyPI on `release` and to TestPyPI on `push`.